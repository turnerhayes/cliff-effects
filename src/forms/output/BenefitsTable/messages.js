export default {
  currentSNAP: { id: 'visitPage.predictions.benefitsTable.currentSNAP' },
  
  futureSNAP: { id: 'visitPage.predictions.benefitsTable.futureSNAP' },
  
  SNAPDiff: { id: 'visitPage.predictions.benefitsTable.SNAPDiff' },
  
  currentSection8: { id: 'visitPage.predictions.benefitsTable.currentSection8' },
  
  futureSection8: { id: 'visitPage.predictions.benefitsTable.futureSection8' },
  
  section8Diff: { id: 'visitPage.predictions.benefitsTable.section8Diff' },
  
  currentTotalBenefits: { id: 'visitPage.predictions.benefitsTable.currentTotalBenefits' },
  
  futureTotalBenefits: { id: 'visitPage.predictions.benefitsTable.futureTotalBenefits' },
  
  totalBenefitsDiff: { id: 'visitPage.predictions.benefitsTable.totalBenefitsDiff' },
  
  currentIncome: { id: 'visitPage.predictions.benefitsTable.currentIncome' },
  
  futureIncome: { id: 'visitPage.predictions.benefitsTable.futureIncome' },
  
  incomeDiff: { id: 'visitPage.predictions.benefitsTable.incomeDiff' },
  
  currentNetTotal: { id: 'visitPage.predictions.benefitsTable.currentNetTotal' },
  
  futureNetTotal: { id: 'visitPage.predictions.benefitsTable.futureNetTotal' },
  
  netTotalDiff: { id: 'visitPage.predictions.benefitsTable.netTotalDiff' },
  
  rows: {
    SNAP: { id: 'visitPage.predictions.benefitsTable.rows.SNAP' },
    
    section8: { id: 'visitPage.predictions.benefitsTable.rows.section8' },
    
    totalBenefits: { id: 'visitPage.predictions.benefitsTable.rows.totalBenefits' },
    
    income: { id: 'visitPage.predictions.benefitsTable.rows.income' },
    
    netTotal: { id: 'visitPage.predictions.benefitsTable.rows.netTotal' },
  },
  
  columnHeaders: {
    benefit: { id: 'visitPage.predictions.benefitsTable.columnHeaders.benefit' },

    currentBenefits: { id: 'visitPage.predictions.benefitsTable.columnHeaders.currentBenefits' },

    newEstimate: { id: 'visitPage.predictions.benefitsTable.columnHeaders.newEstimate' },

    difference: { id: 'visitPage.predictions.benefitsTable.columnHeaders.difference' },
  },
};
