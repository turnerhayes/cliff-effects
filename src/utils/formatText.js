import React from 'react';
import marksy from 'marksy';

const compile = marksy({ createElement: React.createElement });

const formatText = ({ text }) => {
  return compile(text).tree;
};

export default formatText;
