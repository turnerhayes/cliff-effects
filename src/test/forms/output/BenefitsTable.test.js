import React from 'react';
import renderer from 'react-test-renderer';
import { cloneDeep, set } from 'lodash';

import BenefitsTable from '../../../forms/output/BenefitsTable';
import { CLIENT_DEFAULTS } from '../../../utils/CLIENT_DEFAULTS';
import { wrapWithIntlProvider } from '../../utils/test-utils';


const buildSnapshot = (client) => {
  const rendered = renderer.create(
    wrapWithIntlProvider(
      <BenefitsTable 
        client={ client } />
    )
  );

  return rendered.toJSON();
};

test('Benefits table renders correctly', () => {
  const client = cloneDeep(CLIENT_DEFAULTS);
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'current.hasSnap', true);
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'current.earned', 100);
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'future.earned', 200);
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'current.earned', 300);
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'current.hasSection8');
  expect(buildSnapshot(client)).toMatchSnapshot();

  set(client, 'future.earned', 400);
  expect(buildSnapshot(client)).toMatchSnapshot();
});
