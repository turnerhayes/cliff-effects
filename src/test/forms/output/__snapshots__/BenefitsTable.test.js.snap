// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefits table renders correctly 1`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 2`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 3`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $100 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $-100 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $292 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 4`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $100 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $200 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $292 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $392 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 5`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $300 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $200 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $-100 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $492 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $392 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 6`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $0 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $300 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $200 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $-100 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $492 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $392 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Benefits table renders correctly 7`] = `
<div>
  <table
    className="ui celled table"
  >
    <thead
      className=""
    >
      <tr
        className=""
      >
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Benefit
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Current Benefits
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            New Estimate
          </span>
        </td>
        <td
          className="three wide"
          style={
            Object {
              "background": "rgba(0, 181, 173, 1)",
              "borderRadius": "inherit",
              "color": "white",
              "fontSize": "1.3em",
              "fontWeight": 900,
              "letterSpacing": "0.02em",
              "textAlign": "center",
            }
          }
        >
          <span>
            Difference
          </span>
        </td>
      </tr>
    </thead>
    <tbody
      className=""
    >
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            SNAP
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $192 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $187 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            -$5 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
      >
        <td
          className=""
          style={
            Object {
              "fontSize": "1.1em",
              "fontWeight": 700,
              "textAlign": "left",
            }
          }
        >
          <span>
            Income
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $300 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $400 / month
          </span>
        </td>
        <td
          className="right aligned"
        >
          <span>
            $100 / month
          </span>
        </td>
      </tr>
      <tr
        className=""
        style={
          Object {
            "border": "none",
          }
        }
      >
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.2em",
              "fontWeight": 700,
              "padingTop": "0.25em",
              "textAlign": "left",
            }
          }
        >
          <span>
            Net Total
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $492 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $587 / month
          </span>
        </td>
        <td
          className="right aligned three wide"
          style={
            Object {
              "borderTop": "2px solid rgba(0, 181, 173, 1)",
              "fontSize": "1.1em",
              "fontWeight": 700,
              "padingTop": "0.25em",
            }
          }
        >
          <span>
            $95 / month
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
